From 7dbd8cb759c9873c3f624ceed894fb47ff869b7d Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Wed, 6 Nov 2013 12:36:03 +0100
Subject: monitor: Remove usb_add/del commands for Red Hat Enterprise Linux

RH-Author: Miroslav Rezanina <mrezanin@redhat.com>
Message-id: <daf2ca0042519cd01a6a68d30eb76bdd8b88c9f2.1383741033.git.mrezanin@redhat.com>
Patchwork-id: 55520
O-Subject: [RHEL7 qemu-kvm PATCH v2 3/4] monitor: Remove usb_add/del commands for Red Hat Enterprise Linux
Bugzilla: 1010858
RH-Acked-by: Michal Novotny <minovotn@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Eric Blake <eblake@redhat.com>

From: Miroslav Rezanina <mrezanin@redhat.com>

This is forwardport of RHEL-6 commit 754e6292b4ab41c8848171555af830ab7284f4d3:

    monitor: Remove usb_add/del commands for Red Hat Enterprise Linux

    The usb_add/del commands have been obsoleted by the device_add/del
    commands.

    Signed-off-by: Amit Shah <amit.shah@redhat.com>

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hmp-commands.hx | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hmp-commands.hx b/hmp-commands.hx
index 450f1fb..5011593 100644
--- a/hmp-commands.hx
+++ b/hmp-commands.hx
@@ -646,6 +646,7 @@ STEXI
 Compute the checksum of a memory region.
 ETEXI
 
+#if 0 /* Disabled for Red Hat Enterprise Linux */
     {
         .name       = "usb_add",
         .args_type  = "devname:s",
@@ -676,6 +677,7 @@ Remove the USB device @var{devname} from the QEMU virtual USB
 hub. @var{devname} has the syntax @code{bus.addr}. Use the monitor
 command @code{info usb} to see the devices you can remove.
 ETEXI
+#endif
 
     {
         .name       = "device_add",
-- 
2.5.5

